config/packagebundlecontroller/pbc.yaml.tmp

```diff
@@ -0,0 +1,12 @@
+apiVersion: packages.eks.amazonaws.com/v1alpha1
```

Comment:
Can we use the helm chart for this and pass in values.yaml with the image tages?